sinonjs/samsam
# Import match from sinon

This PR is a ⚠️ work in progress ⚠️, and is a partial solution to https://github.com/sinonjs/referee/issues/29

The imported `sinon-match` is not exported yet, this is just to make sure we can actually import it and run the tests.

#### How to verify - mandatory

1. In your `@sinonjs/commons` clone, checkout https://github.com/sinonjs/commons/pull/3
1. `npm link`
1. In your `@sinonjs/samsam` clone, check out this branch
1. `npm install`
1. `npm link @sinonjs/commons`
1. `npm test`
1. Observe that there's just one failing test
1. Soon™️

#### Checklist for author
